### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by jnyrup**
Updates:
* .net core 3.0-RC1
* test packages
* project system guid

remove binary git files, now fetched via `GitVersion.CommandLine.DotNetCore`

### CODE REVIEW

File: Tests/NetCore.Specs/NetCore.Specs.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Chill" Version="4.0.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```
Comment (lg2de) [inline Tests/NetCore.Specs/NetCore.Specs.csproj]: In Tests/Net47.Specs/Net47.Specs.csproj you referenced 16.3.0.

Comment (jnyrup) [reply Tests/NetCore.Specs/NetCore.Specs.csproj]: Well spotted 👍 
For some reason (probably a mistake), 16.3.0 requires netcoreapp2.1.
I've reported it [here](https://github.com/microsoft/vstest/issues/2188)

For the TestFrameworks I upgraded to netcoreapp2.1, for the tests of the core library I updated to the currently latest supported version of `Microsoft.NET.Test.Sdk`.

**Comment by danielmpetrov** [comment]
> .net core 3.0-RC1

@jnyrup .NET Core 3.0 is launching at [.NET Conf](https://www.dotnetconf.net/) 2019 in 2 days, maybe you can wait for that 🎉

**Comment by jnyrup** [comment]
@dennisdoomen Will you edit the AppVeyor script? It tries to invoke `\Lib\GitVersion.exe`

**Comment by dennisdoomen** [comment]
>  Will you edit the AppVeyor script? It tries to invoke \Lib\GitVersion.exe

What can I call instead? Just remove it?

**Comment by jnyrup** [comment]
It should just be deleted. (it looks like it also has some psake leftovers?)
The output nupkg from the build has the expected Assembly Versions on the dll's.

**Comment by dennisdoomen** [comment]
I used it to update the AppVeyor build number, but maybe there's a better way to do that.

**Comment by dennisdoomen** [comment]
Done

**Pull request merged**